src/io/flutter/inspector/InspectorTree.java

```diff
@@ -93,9 +93,6 @@ public InspectorTree(final DefaultMutableTreeNode treemodel,
     super(treemodel);
     setUI(new InspectorTreeUI());
     final BasicTreeUI ui = (BasicTreeUI)getUI();
-    if (!legacyMode) {
-      setBackground(VERY_LIGHT_GREY);
```

Comment:
Any objections if I instead move this as an option. I think we will at least want to try having it on in some user studies to see whether it helps users understand the context for the details tree. the challenge is sometimes when you change selection in the master tree we don't want to change the root of the details tree and seeing this rectangle helps explain that.

Reply:
Just for context, the visual artifact I was trying to address here was that the details area often looked like it wasn't taking the full scrollable space in the scrollpane. This would happen when the details didn't have much content - the bk color of the details would be white, the bk color of the scrollpane would be light gray, and it looked like the details just weren't stretched to fill the available space.

Reply:
No objection to an option, though I don't know where this would be displayed. I'll retain the code w/ a TODO:.